docs/server/source/quickstart.md

```diff
@@ -1,6 +1,12 @@
 # Quickstart
 
-This page has instructions to set up a single stand-alone BigchainDB node for learning or experimenting. Instructions for other cases are [elsewhere](introduction.html). We will assume you're using Ubuntu 16.04 or similar. If you're not using Linux, then you might try [running BigchainDB with Docker](appendices/run-with-docker.html).
+This page has instructions to set up a single stand-alone BigchainDB node manually for learning or experimenting. Instructions for other cases are [elsewhere](introduction.html). You might try one of the following deployment methods
+as well, depending on your choice:
+- [BigchainDB with Docker](appendices/run-with-docker.html).
```

Comment:
I think a "Quickstart" page should have no steps where the reader must make a decision. A Quickstart page should be "on rails." This PR adds a four-option decision first thing on the Quickstart page. Let's remove that.

The sentence "Instructions for other cases are elsewhere." already links to the Introduction page, and that page has "Setup Instructions for Various Cases". Let's put links to the various other setup options there. We could maybe organize those links differently.

What are all these different options for? Help the reader decide which one is for them. Which ones are just for trying BigchainDB? Which ones are for being a developer of BigchainDB Server software? How would you categorize these options:

* Run BigchainDB with Docker
* Run BigchainDB with Docker on Mac
* Run BigchainDB with Vagrant
* Run BigchainDB with Ansible

Reply:
While you're editing the Introduction page:

I noticed that one of the links on the Introduction page is broken (because a previous PR deleted the page that it linked to). That link can be removed. It's the one with text, "Deploy a bare-bones RethinkDB-based node on any Ubuntu machine with Ansible". That whole line can be deleted.

Reply:
Removed the broken link. For the other comment ,I can think of two options.
- We can change the `quickstart` in a way that we specify all these ways to install and the user can pick whatever they want to choose i.e. the new `quickstart` page can look like this:
```text
  - Run BigchainDB on Host(Ubuntu 16.04)
  - Run BigchainDB with Docker
  - Run BigchainDB with Vagrant
  - Run BigchainDB with Ansible
```

OR

- We don't change the instructions on the `quickstart` page and change the `introduction` page to something like this.
```text
....
  - Set up and run a local dev/test node for developing and testing BigchainDB Server.
    - Run BigchainDB on Host/Local Dev.
    - Run BigchainDB with Docker.
    - Run BigchainDB with Vagrant.
    - Run BigchainDB with Ansible.
...
```
Instead of `setup-run-node` having OptionA,B..., we can just redirect from the introduction page to the relevant installation. Comments? But option 2 will need some refactoring in terms of files and updating the `setup-run-node` which I think might be broken. @ttmc

Reply:
Maybe it would be better to move the list of all the installation options out of the "Introduction" page and into a new page titled "Install BigchainDB". That would be better for search engines as well.

I definitely prefer to keep the "Quickstart" page in its own little world, not asking the reader to make any decisions.

I know it would take time, but it would be a good idea to refactor `setup-run-node`. Maybe into separate pages? In fact, all the various installation methods (pages) could be moved from the Appendices into their own section.

Reply:
This makes sense. How about we merge this PR as is, if you don't have any comments on the  rest of the components and I will issue another PR for the docs refactor.

Reply:
On second thought, refactoring the installation-related docs should be done in a separate PR. This PR is already big enough.

Reply:
So should I take the docs changes out of this PR? And make it part of another one?